packages/smart-contracts/scripts-create2/contract-setup/adminTasks.ts

```diff
@@ -11,88 +13,99 @@ export const BATCH_FEE = 3;
 export const updateChainlinkConversionPath = async (
   contract: any,
   network: string,
-  nonce: number,
   gasPrice: BigNumber,
+  version = undefined,
 ): Promise<void> => {
   const currentChainlinkAddress = await contract.chainlinkConversionPath();
-  const chainlinkConversionPathAddress = chainlinkConversionPath.getAddress(network, '0.1.0');
+  const chainlinkConversionPathAddress = chainlinkConversionPath.getAddress(network, version);
   if (currentChainlinkAddress !== chainlinkConversionPathAddress) {
-    await contract.updateConversionPathAddress(chainlinkConversionPathAddress, {
-      nonce: nonce,
+    const tx = await contract.updateConversionPathAddress(chainlinkConversionPathAddress, {
       gasPrice: gasPrice,
```

Comment:
Note for later: `gasPrice` is legacy and more expensive most of the time, we should migrate to EIP-1559 gas fee.

Reply:
I'll create a tech discovery ticket to not forget about that